herbert_core/utilities/classes/@object_lookup/private/parse_sz_repmat.m

```diff
@@ -0,0 +1,57 @@
+function sz_repmat_out = parse_sz_repmat (sz_repmat, n_objArr, ntot_objArr)
+% Check that the repmat option for implicitly expanding the stored arrays in an
+% object_lookup object is valid. Throw an error if not.
+%
+%   >> sz_repmat_out = parse_sz_repmat (sz_repmat, n_objArr, ntot_objArr)
```

Comment:
I do not see what benefit this function adds beyond what repmat does.

Reply:
See above. The point about object_lookup is to only keep the minimum storage.